webapp/src/views/MqttAdminView.vue

```diff
@@ -80,7 +80,7 @@
                     v-model="mqttConfigList.mqtt_publish_interval"
                     type="number"
                     min="5"
-                    max="86400"
+                    max="65535"
```

Comment:
I updated this value because I saw the [correct value in the firmware](https://github.com/tbnobody/OpenDTU/blob/4cd5d79c73af24db5f547cbe944d9f8105b11798/src/WebApi_mqtt.cpp#L247) and when copying UI elements I noticed this value being incorrect.